pkgs/tools/text/tab/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromBitbucket, python2 }:
+
+stdenv.mkDerivation rec {
+  version = "7.1";
+  pname = "tab";
+  name = "${pname}-${version}";
+
+  src = fetchFromBitbucket {
+    owner = "tkatchev";
+    repo = "${pname}";
+    rev = "${version}";
+    sha256 = "049wx24b4v8alhvc05z7snyzaf6fk9w2npc4kklp366yjl1l4qbb";
+  };
+
+  buildInputs = [ python2 ];
+
+  doCheck = true;
+
+  checkPhase = ''make test'';
+
+  installPhase = ''
```

Comment:
1. missing "runHook preInstall" and postInstall
2. the 2 lines can be combined to ```install -Dm555 -t $out/bin tab```
3. there seems to be docs the source - please install those too

Reply:
1. Added hooks. Thanks.
2. Yes, 2 lines can be combined,  what would be an advantage of that solution, besides saving one line :) ?
3. Yes, there are docs, but "make install" does not install them, and also [installation instruciton](https://tkatchev.bitbucket.io/tab/docs.html#compiling-and-installing) says to just copy the excutable. Man page would be useful, but there isn't one. There is quite extensive help from the excutable though (`tab -h`).

Reply:
Regd docs, please copy them in. It's nicer for the end user if there is something more than just a binary regardless of what upstream installs.

Reply:
> Yes, 2 lines can be combined, what would be an advantage of that solution, besides saving one line :) ?

```install``` allows you to create the directory, copy files, set permissions and set ownership all in one go which makes it much easier to read in general (not just in the context of nix) as it keeps everything that touches a certain file or set of files together in one place as opposed to spread out.